lib/content-services/src/lib/search/components/search-filter-autocomplete-chips/search-filter-autocomplete-chips.component.ts

```diff
@@ -120,6 +125,9 @@ export class SearchFilterAutocompleteChipsComponent implements SearchWidget, OnI
         if (this.settings.field === AutocompleteField.CATEGORIES && value) {
             this.searchForExistingCategories(value);
         }
+        if (this.settings.field === AutocompleteField.LOCATION && value) {
```

Comment:
value condition is repeated in above if. Can we split those conditions to don't repeat them? Something like: 
```ts
if (value) {
  if (this.settings.field === AutocompleteField.CATEGORIES) {
     ...
  } else if (this.settings.field === AutocompleteField.LOCATION) {
     ...
  } 
}
```

I have added outer if to avoid repeating. I also added 'else' because field can't be LOCATION and CATEGORIES in the same time. It can be only one of them so else will be better to avoid checking both